sui_core/src/authority_batch.rs

```diff
@@ -193,6 +194,16 @@ impl crate::authority::AuthorityState {
                     .batches
                     .insert(&new_batch.batch.next_sequence_number, &new_batch)?;
 
```

Comment:
It seems we never create empty batches (line 183), aren't empty batches needed for reconfiguration?

Reply:
Very good question!

Reply:
added an issue about this